### ISSUE TITLE
Weired way of computing logits in model

### ISSUE THREAD

**Issue by fzdy1914** [voiler/IMPALA#3]
I doubt that the logits should be computed as below instead of `logits.view(seq_len, -1, bs)`
```
if not actor:
    return logits.view(seq_len, bs, -1).permute(0, 2, 1), values.view(seq_len, bs)
```

Considering:
`logits = torch.stack(logits).permute(1, 2, 0)`
